### PR TITLE
Iv-analysis

### PR DESCRIPTION
**Pull request by maasencioh**

### CODE REVIEW

**Comment by maasencioh** [comment]
@targos @lpatiny the `title` property that was in `spectrum` was moved to `analysis.label`?

**Comment by targos** [comment]
I can't answer. I have not been involved in writing the code of this library.

**Comment by maasencioh** [comment]
@targos it was before in `common-spectrum`, just in case

File: packages/iv-analysis/package.json [hunk 1]
```diff
@@ -0,0 +1,65 @@
+{
+  "name": "iv-analysis",
+  "version": "0.15.0",
```
Comment (targos) [inline packages/iv-analysis/package.json:3]:
```suggestion
  "version": "0.0.0",
```

File: release-please-config.json [hunk 2]
```diff
@@ -1,8 +1,6 @@
 {
   "bootstrap-sha": "3b7fca13b911d359016ad58693ccfc8f95d29b0c",
-  "plugins": [
-    "node-workspace"
-  ],
+  "plugins": ["node-workspace"],
```
Comment (targos) [inline release-please-config.json]: that's useless, as the script will re-add the new lines next time

Comment (maasencioh) [reply release-please-config.json]: it was from prettier, should we remove this file from prettier review?

Comment (targos) [reply release-please-config.json]: yes

**Review by targos — COMMENTED** [review]
I haven't looked at the implementation. Project setup LGTM

**Pull request merged**